lib/ukrand/Config.uk

```diff
@@ -0,0 +1,56 @@
+menuconfig LIKUKRAND
+	bool "ukrand: Software random number generator"
+	default n
+
+if LIKUKRAND
+
+config LIBUKSWRAND
+	bool "Software-generated randomness"
+	default n
+
+if LIBUKSWRAND
+
+choice
+	prompt "Algorithm"
+	default LIBUKSWRAND_CHACHA
+
+config LIBUKSWRAND_MWC
+	bool "Multiply-with-carry"
+	help
+	  Use multiply-with-carry algorithm
+
+config LIBUKSWRAND_CHACHA
+	bool "ChaCha20"
+	help
+	  Use ChaCha20 algorithm
+endchoice
+
+choice
+	prompt "Initial seed"
+	default LIBUKSWRAND_INITIALSEED_TIME
+
+config LIBUKSWRAND_INITIALSEED_TIME
```

Comment:
Timestamp and constant seeds are not secure, so these options must be deprecated too. In fact they are even worse than not secure as they give a false sense of security. Right now the only meaningful option is processor generated randomness, so it's either that the hardware that or nothing. In the future, as we support more TRNGs we can add more here.